pkg/logging/filelogging.go

```diff
@@ -0,0 +1,56 @@
+package logging
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+// LogCreatedFiles checks all the asset files created and logs it for the user to see.
+func LogCreatedFiles(cmdName string, directory string, targets []asset.WritableAsset) {
+	directory = strings.TrimRight(directory, `/`)
+
+	assetDirs := sets.NewString()
+	for _, a := range targets {
+		for _, f := range a.Files() {
+			index := strings.LastIndex(f.Filename, "/")
+			path := directory
+
+			if index != -1 {
+				path += fmt.Sprintf(`/%s`, f.Filename[:index])
+			}
+			assetDirs.Insert(path)
+		}
+	}
+
+	if len(assetDirs) == 0 {
+		return
+	}
+
+	var directories string
+	keys := assetDirs.List()
+
+	if len(keys) == 1 {
+		directories = keys[0]
+	} else {
+		for _, key := range keys[:len(keys)-1] {
+			directories += key + ", "
+		}
+
+		directories = fmt.Sprintf("%s and %s", strings.TrimRight(directories, ", "), keys[len(keys)-1])
+	}
+
+	directories = fmt.Sprintf("%s director", directories)
+
+	if len(assetDirs) == 1 {
+		directories += "y"
+	} else {
+		directories += "ies"
+	}
+
+	logrus.Infof("%s created in the %s", cmdName, directories)
```

Comment:
I don't think this function should be logging directly. just return a string that summarizes what was created/generated given a list of Assets.